### PR TITLE
Remove 'iotjs_jval_set_prototype' to reduce binary size.

### PR DESCRIPTION
**Pull request by LaszloLango**
IoT.js-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
The `.text` section is reduced by 156 bytes.

File: src/modules/iotjs_module_fs.c [hunk 1]
```diff
@@ -41,7 +41,61 @@ static void iotjs_fs_reqwrap_destroy(iotjs_fs_reqwrap_t* fs_reqwrap) {
...
+JS_FUNCTION(StatsIsFile) {
+  DJS_CHECK_THIS(object);
+  iotjs_jval_t stats = JS_GET_THIS(object);
+  return StatsIsTypeOf(stats, S_IFREG);
+}
+
+
+static iotjs_jval_t MakeStatObject(uv_stat_t* statbuf) {
+  iotjs_jval_t jstat = iotjs_jval_create_object();
+  iotjs_jval_t stat_prototype = iotjs_jval_create_object();
```
Comment (galpeter) [inline src/modules/iotjs_module_fs.c]: With this approach we always recreate the Stat object prototype and each time it is a different one. So we don't get anything with using prototypes as two stat objects prototype will be different thus there isn't any sense to have a prototype object at all.

The old way was to ensure that every Stat object will have the same prototype.

**Comment by LaszloLango** [comment]
@galpeter thanks for your review. I've update the PR.

**Comment by glistening** [comment]
@LaszloLango I would like to keep `iotjs_jval_set_prototype`.

i) The binary size gain came from the fact that the only usage of `iotjs_jval_set_prototype` does not handle error case. If we add the checking error (it should be there), perhaps there are no binary gain.

ii) Although it may be transient, for setting method, I have to use `iotjs_jval_set_method`. For property, I have to use `jerry_set_prototype`. If we will keep `iotjs_jval_set_method`, `iotjs_jval_set_property_jval`, ....  ( I am voting for keep since they are handy to use.), I would like to keep `iotjs_jval_set_prototype`.

**Comment by LaszloLango** [comment]
@glistening, OK. I don't insist on this PR. Unfortunately my initial idea did not work well, so I had to reduce the patch to this version. The PR does not have a significant gain, so let's close it.

**Pull request closed without merging**